src/vivarium_public_health/risks/data_transformations.py

```diff
@@ -83,33 +77,21 @@ def post_processor(exposure, _):
     return post_processor
 
 
-def load_distribution_data(builder: Builder, risk: EntityString) -> Dict[str, Any]:
-    distribution_type = get_distribution_type(builder, risk)
-    exposure_data, _ = get_exposure_data(builder, risk, distribution_type)
+def load_distribution_data(builder: Builder, risk: "Risk") -> Dict[str, Any]:
+    distribution_type = risk.distribution_type
```

Comment:
nit: just use risk.distribution_type in the calls (like how you are passing risk.risk)

Reply:
This will all go away or get improved in a later PR.